src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -43,51 +50,91 @@ REACT_NATIVE_XCODE="../node_modules/react-native/scripts/react-native-xcode.sh"
+export SENTRY_CLI_EXECUTABLE="paht/to/@sentry/cli/bin/sentry-cli"
+export SENTRY_CLI_EXTRA_ARGS="--extra --flags" # Extra arguments for sentry-cli in all build phases
+export SENTRY_CLI_RN_XCODE_EXTRA_ARGS="--extra --flags"
+
+export SOURCE_MAP_PATH="path/to/source-maps" # From where collect-modules should read modules
+export SENTRY_COLLECT_MODULES="path/to/collect-modules.sh"
+export MODULES_PATHS="../node_modules,../my-custom-module"
 ```
 
-Additionally, we add a build script named “Upload Debug Symbols to Sentry” to upload debug symbols to Sentry.
+By default, uploading of source maps for debug simulator builds is disabled for speed reasons. If you want to generate source maps for debug builds, you can pass `--allow-fetch` as a parameter to `SENTRY_CLI_RN_XCODE_EXTRA_ARGS` in the above mentioned script.
 
-#### Upload Debug Symbols to Sentry
+### Add auto debug symbols upload
 
-To upload source maps and symbols to Sentry, create a new Run Script build phase using the following script:
+To upload debug symbols to Sentry, create a new Run Script build phase using the following script:
 
-```bash
+```bash {filename:Upload Debug Symbols to Sentry}
 export SENTRY_PROPERTIES=sentry.properties
 
 [[ $SENTRY_INCLUDE_NATIVE_SOURCES == "true" ]] && INCLUDE_SOURCES_FLAG="--include-sources" || INCLUDE_SOURCES_FLAG=""
 SENTRY_CLI="../node_modules/@sentry/cli/bin/sentry-cli"
 $SENTRY_CLI debug-files upload "$INCLUDE_SOURCES_FLAG" "$DWARF_DSYM_FOLDER_PATH"
 ```
 
-By default, uploading of debug simulator builds is disabled for speed reasons. If you want to generate debug symbols for debug builds, you can pass `--allow-fetch` as a parameter to `react-native-xcode` in the above mentioned build phase.
+To cahnge the default behavior, you can pass the following environment variables in `.xcode.env` or in the Build Phase script:
+
+```bash {filename:.xcode.env}
+export SENTRY_PROPERTIES=path/to/sentry.properties
+export SENTRY_DISABLE_AUTO_UPLOAD=true # Temporarily disable debug symbols upload
+
+export SENTRY_INCLUDE_NATIVE_SOURCES=true # Upload native iOS sources
+export SENTRY_CLI_EXECUTABLE="paht/to/@sentry/cli/bin/sentry-cli"
+export SENTRY_CLI_EXTRA_ARGS="--extra --flags" # Extra arguments for sentry-cli in all build phases
+export SENTRY_CLI_DEBUG_FILES_UPLOAD_EXTRA_ARGS="--extra --flags"
+```
+
+For more information about what options to use in `SENTRY_CLI_DEBUG_FILES_UPLOAD_EXTRA_ARGS` visit [Sentry CLI Apple Debug Symbols](/platforms/apple/dsym/#sentry-cli) documentation.
+
+### Linking the Native Library via CocoaPods
+
+To link the Native library via CocoaPods, link to the Native module in your Podfile. In React Native versions 0.60 and above, the podspec should be automatically linked, if that's not the case, you can link to the native module manually by adding the `RNSentry.podspec` to your Podfile. See below:
 
-For more information visit [Sentry CLI Apple Debug Symbols](/platforms/apple/dsym/#sentry-cli) documentation.
+```rb
```

Comment:
I think "rb" isn't recognized by our syntax highlighter, but "ruby" should be

```suggestion
```ruby
```